pkg/api/config.go

```diff
@@ -211,6 +211,24 @@ func searchForTestDuplicates(tests []TestStepConfiguration) []error {
 	return nil
 }
 
+func searchForTestConfigDuplicates(tests []TestConfig) []error {
```

Comment:
The new test type should be an optional field under the `TestStepConfiguration`, so this function will be redundant with `searchForTestDuplicates`